DependencyInjection/SonataPageExtension.php

```diff
@@ -54,6 +54,8 @@ public function load(array $configs, ContainerBuilder $container)
             if (!$config['direct_publication']) {
                 $container->removeDefinition('sonata.page.admin.extension.snapshot');
             }
+
+            $this->configureTemplatesAdmin($container, $config);
```

Comment:
I'm not really sure this is BC break.

If the admin-bundle is not loaded, who the f*ck will use admin templates? :-)

cc @sonata-project/contributors